app/src/main/res/navigation/main_navigation.xml

```diff
@@ -309,8 +306,23 @@
         <action
             android:id="@+id/action_settings_to_dataManagementSettings"
             app:destination="@id/dataManagementSettingsFragment" />
+        <action
+            android:id="@+id/action_settings_to_externalContentSetting"
+            app:destination="@id/externalContentSettingFragment" />
+        <action
+            android:id="@+id/action_settingsFragment_to_aiEngineSettingFragment"
+            app:destination="@id/aiEngineSettingFragment" />
```

Comment:
This direction already exists and is better placed, so remove the one that's been added with your PR